lib/index.js

```diff
@@ -14,13 +14,14 @@ VM.deps = {
   ethUtil: require('ethereumjs-util'),
   Account: require('ethereumjs-account'),
   Trie: require('merkle-patricia-tree'),
-  rlp: require('ethereumjs-util').rlp
+  rlp: require('ethereumjs-util').rlp,
+  Blockchain: require('ethereumjs-blockchain')
```

Comment:
Whoops, I was trying to be fancy with something that I eventually removed. Will remove it.